compiler/rustc_data_structures/src/aligned.rs

```diff
@@ -15,10 +13,10 @@ pub const fn align_of<T: ?Sized + Aligned>() -> Alignment {
 /// # Safety
 ///
 /// `Self::ALIGN` must be equal to the alignment of `Self`. For sized types it
-/// is [`mem::align_of<Self>()`], for unsized types it depends on the type, for
+/// is [`align_of<Self>()`], for unsized types it depends on the type, for
```

Comment:
shouldn't this be turbofished?

Reply:
I've fixed this and others now in https://github.com/rust-lang/rust/pull/138034/commits/88cec8c4956c244381c916a847776cbe01979b10.